api/requirements.txt

```diff
@@ -0,0 +1,3 @@
+redis==2.10.3
```

Comment:
It sure does, it would just be optimal that it was in sync with `setup.py` - see reference code above.